indexer/src/main/java/au/org/aodn/esindexer/model/RelationType.java

```diff
@@ -5,6 +5,9 @@
 @Getter
 public enum RelationType {
     SELF("self"),
+    PARENT("parent"),
+    SIBLING("sibling"),
+    CHILD("child"),
```

Comment:
why not adding "brothersAndSisters" to here? so you can consistently use `RelationType` enum with your `getRecordMaps` currently uses "String key" (getRecordMaps still need to be renamed as commented)

Reply:
The `RelationType` enum contains recommended values from the [stac collection ](https://github.com/radiantearth/stac-spec/blob/master/collection-spec/collection-spec.md#relation-types), where as the "brotherAndSisters" (also including "children") are `key` in a `map` from the geonetwork api. "parent" is the same, but "children" and "brothersAndSisters" are not good values of Relation Type in stac collection specification. I don't want to confuse others (someone may use "brothersAndSisters" in `rel` field of stac collection, which is bad). So i didn't put them all into the `RelationType` enum, and just use `key` for the relation of geonetwork api, to differentiate with the `RelationType` in stac collection.